<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cantina</class>
 <widget class="QMainWindow" name="Cantina">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>769</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>637</width>
    <height>769</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MAPPA VASI VINARI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sfondoCantina.qrc">
    <normaloff>:/sfondo/2_grappoloicon.png</normaloff>:/sfondo/2_grappoloicon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="mappacantina">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/sfondo/3_Planimetria_Botti.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QDateTimeEdit" name="dataora">
      <property name="minimumSize">
       <size>
        <width>496</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 236, 255);</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="lista">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>102</width>
         <height>1101</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="vv16">
          <property name="text">
           <string>v.v. 16</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="locale2">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(221, 237, 255);</string>
          </property>
          <property name="text">
           <string>LOCALE 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="vv14">
          <property name="text">
           <string>v.v. 14</string>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QPushButton" name="vv22">
          <property name="text">
           <string>v.v. 22</string>
          </property>
         </widget>
        </item>
        <item row="32" column="0">
         <widget class="QPushButton" name="vv33">
          <property name="text">
           <string>v.v. 33</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="vv7">
          <property name="text">
           <string>v.v. 7</string>
          </property>
         </widget>
        </item>
        <item row="31" column="0">
         <widget class="QPushButton" name="vv32">
          <property name="text">
           <string>v.v. 32</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="vv19">
          <property name="text">
           <string>v.v. 19</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="vv15">
          <property name="text">
           <string>v.v. 15</string>
          </property>
         </widget>
        </item>
        <item row="26" column="0">
         <widget class="QPushButton" name="vv27">
          <property name="text">
           <string>v.v. 27</string>
          </property>
         </widget>
        </item>
        <item row="24" column="0">
         <widget class="QPushButton" name="vv25">
          <property name="text">
           <string>v.v. 25</string>
          </property>
         </widget>
        </item>
        <item row="22" column="0">
         <widget class="QPushButton" name="vv23">
          <property name="text">
           <string>v.v. 23</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QPushButton" name="vv8">
          <property name="text">
           <string>v.v. 8</string>
          </property>
         </widget>
        </item>
        <item row="28" column="0">
         <widget class="QPushButton" name="vv29">
          <property name="text">
           <string>v.v. 29</string>
          </property>
         </widget>
        </item>
        <item row="29" column="0">
         <widget class="QPushButton" name="vv30">
          <property name="text">
           <string>v.v. 30</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="vv17">
          <property name="text">
           <string>v.v. 17</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QPushButton" name="locale3">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(221, 237, 255);</string>
          </property>
          <property name="text">
           <string>LOCALE 3</string>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QPushButton" name="vv28">
          <property name="text">
           <string>v.v. 28</string>
          </property>
         </widget>
        </item>
        <item row="23" column="0">
         <widget class="QPushButton" name="vv24">
          <property name="text">
           <string>v.v. 24</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="vv20">
          <property name="text">
           <string>v.v. 20</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="vv18">
          <property name="text">
           <string>v.v. 18</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="vv21">
          <property name="text">
           <string>v.v. 21</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QPushButton" name="vv10">
          <property name="text">
           <string>v.v. 10</string>
          </property>
         </widget>
        </item>
        <item row="30" column="0">
         <widget class="QPushButton" name="vv31">
          <property name="text">
           <string>v.v. 31</string>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QPushButton" name="vv26">
          <property name="text">
           <string>v.v. 26</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QPushButton" name="vv11">
          <property name="text">
           <string>v.v. 11</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QPushButton" name="vv12">
          <property name="text">
           <string>v.v. 12</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QPushButton" name="vv13">
          <property name="text">
           <string>v.v. 13</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPushButton" name="vv6">
          <property name="text">
           <string>v.v. 6</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="vv4">
          <property name="text">
           <string>v.v. 4</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QPushButton" name="vv5">
          <property name="text">
           <string>v.v. 5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="locale1">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(221, 237, 255);</string>
          </property>
          <property name="text">
           <string>LOCALE 1</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QPushButton" name="vv9">
          <property name="text">
           <string>v.v. 9</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sfondoCantina.qrc"/>
 </resources>
 <connections/>
</ui>
